### DISCUSSION TITLE
Dropping support for Ruby Sass and LibSass

### DISCUSSION THREAD

**Discussion by 36degrees** [alphagov/govuk-frontend#2632] [Announcements]
Later this year, the Design System team are planning to drop our support for LibSass and Ruby Sass.

Dropping support matches up with the direction the industry is taking and will enable us to [move to the Sass module system](https://github.com/alphagov/govuk-frontend/issues/1791).

If your team uses either [LibSass](https://sass-lang.com/blog/libsass-is-deprecated) (used by node-sass and sassc, among others) or [Ruby Sass](https://sass-lang.com/blog/ruby-sass-is-deprecated), you should get ready to adopt [Dart Sass](https://sass-lang.com/install) instead, as it's the only implementation we'll be supporting.

We'll tell you as soon as we know the date we'll be dropping support. For now, we just want you to have enough time to make any changes you need to. We know that switching to Dart Sass might not be easy for all teams.

If you have any questions, please comment on this post or contact us at [govuk-design-system-support@digital.cabinet-office.gov.uk](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk).

**Comment by querkmachine** [comment]
As a bit of an informal update, we didn't end up dropping LibSass and Ruby Sass in 2022, and we are unlikely to do so before the end of 2023. 

Our original timeline was based on the timeline set by the Sass development team for when they would deprecate and remove the `@import` syntax from Sass. This deprecation has been delayed multiple times, most recently as they waited for Dart Sass to reach 80% adoption (based on npm downloads). That milestone was reached earlier in 2023, and [although plans to deprecate are now being fleshed out](https://github.com/sass/sass/issues/3513), as of writing, there is no set date for when it will happen.

We're now targeting migration to the Sass module system (and subsequently, dropping support for Ruby Sass and LibSass) for [version 6 of Frontend](https://github.com/alphagov/govuk-frontend/milestone/51).

**Comment by 36degrees** [comment]
We dropped support for Ruby Sass and LibSass in v6.0.0
